scripts/run_codeql_pack_validation.sh

```
@@ -0,0 +1,112 @@
+
+run_fixture() {
+    local name="$1" language="$2" source_root="$3"
+    local query_suite="$4"
+    local extra_args="${5:-}"
+
+    local db="$DB_DIR/$name"
+    local sarif="$RESULTS_DIR/$name.sarif"
+
+    echo "==> [$name] creating database (language=$language, source-root=$source_root)"
+    rm -rf "$db"
+    # shellcheck disable=SC2086 -- extra_args is a deliberately-split arg list
```

Shellcheck (0.11.0)

[error] Couldn't parse this shellcheck directive. Fix to allow more checks.

(SC1073)